app/src/main/java/fr/free/nrw/commons/utils/StringUtils.java

```diff
@@ -0,0 +1,23 @@
+package fr.free.nrw.commons.utils;
+
+/**
+ * Utilities class for miscellaneous strings
+ */
+public class StringUtils {
```

Comment:
The code looks awesome in general, thanks @CAVillalobos ! Just renaming this class as LangCodeUtils would be beneficial. Since this is not a regular string operation like trim, copy etc.

Reply:
Sorry for the delay. The class as been renamed and I also took the chance to change a String concatenation to use `String.format` instead of just adding the Strings.